yarn.lock

```diff
@@ -5023,6 +5023,10 @@ safe-buffer@^5.0.1, safe-buffer@~5.0.1:
   version "5.0.1"
   resolved "https://registry.yarnpkg.com/safe-buffer/-/safe-buffer-5.0.1.tgz#d263ca54696cd8a306b5ca6551e92de57918fbe7"
 
+sdp@^1.5.0:
```

Comment:
As far as I can see it's a dependency for `webrtc-adapter`. I don't remember adding a new package, just running `yarn install`